pkg/apis/serving/v1alpha1/kfservice_types.go

```diff
@@ -21,10 +18,23 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+// TFExampleKFService provides an example to the reader and may also be used by tests
+var TFExampleKFService = &KFService{
+	ObjectMeta: metav1.ObjectMeta{
+		Name:      "foo",
+		Namespace: "default",
+	},
+	Spec: KFServiceSpec{
+		Default: ModelSpec{
+			Tensorflow: &TensorflowSpec{ModelURI: "gs://testbucket/testmodel"},
+		},
+	},
+}
+
 // KFServiceSpec defines the desired state of KFService
 type KFServiceSpec struct {
-	MinReplicas int32 `json:"minReplicas"`
-	MaxReplicas int32 `json:"maxReplicas"`
+	MinReplicas int `json:"minReplicas"`
+	MaxReplicas int `json:"maxReplicas"`
```

Comment:
Any reason why min/max replicas are required fields? Looks in order to have auto scale to zero user has to explicit set min=max=0?

Reply:
No. These should be optional. Let's cleanup in another.